### PR TITLE
AA-723: scss fixes

### PR DESCRIPTION
**Pull request by ciduarte**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -362,7 +362,7 @@
 }
 
 .locked-overlay {
-  opacity: 30%;
+  opacity: .3;
```
Comment (ciduarte) [inline src/index.scss:365]: silly me

File: src/course-home/progress-tab/grades/course-grade/GradeBar.scss [hunk 2]
```diff
@@ -26,7 +26,7 @@
 }
 
 .arrow {
-  margin: 0;
+  margin: 0 !important;
```
Comment (ciduarte) [inline src/course-home/progress-tab/grades/course-grade/GradeBar.scss:29]: trying to override some bootstrap stuff that doesn't surface locally, only on prod :/

**Review by mikix — APPROVED** [review]

File: src/course-home/progress-tab/grades/course-grade/GradeBar.scss (hunk 2 above)
Comment (Dillon-Dumesnil) [reply src/course-home/progress-tab/grades/course-grade/GradeBar.scss:29]: Hmm. Minor red flag seeing this, but if you've tried debugging to no avail, I don't have other guesses.

File: src/courseware/course/Sidebar.scss [hunk 3]
```diff
@@ -1,8 +1,3 @@
-@import "~@edx/brand/paragon/fonts";
-@import "~@edx/brand/paragon/variables";
-@import "~@edx/paragon/scss/core/core";
-@import "~@edx/brand/paragon/overrides";
```
Comment (Dillon-Dumesnil) [inline src/courseware/course/Sidebar.scss:4]: Sure?

File: src/index.scss (hunk 1 above)
Comment (Dillon-Dumesnil) [reply src/index.scss:365]: 🤪

**Review by Dillon-Dumesnil — APPROVED** [review]
Sure

File: src/courseware/course/Sidebar.scss (hunk 3 above)
Comment (ciduarte) [reply src/courseware/course/Sidebar.scss:4]: these were being re-imported w/o necessity from some rev work, so just some cleanup

**Pull request merged**